storage/beacondb/registry.go

```diff
@@ -242,3 +243,28 @@ func (kv *KVStore) GetTotalActiveBalances(
 	)
 	return totalActiveBalances, err
 }
+
+// GetPendingPartialWithdrawals is equivalent to `pending_partial_withdrawals`
+// If called before electra, will return an error.
+func (kv *KVStore) GetPendingPartialWithdrawals() ([]*ctypes.PendingPartialWithdrawal, error) {
+	if err := kv.requireEqualOrAfterVersion(version.Electra()); err != nil {
```

Comment:
noting: these version checks are fine as long as we set the Fork object to Electra before state_processing any Electra blocks. May need to be careful of this when proposer building first Electra block.